domains/grid/composables/useGrid.ts

```diff
@@ -0,0 +1,65 @@
+const removeGridLayoutItem = (id: string) => {
+  const { gridLayout, hasUnsavedGrid } = storeToRefs(useAppStore())
+
+  if (!id || typeof id !== 'string') {
```

Comment:
Is it possible for the type of `id` not to be `string` even when argument's type is defined? 
Maybe that's something new for me 🙂

Reply:
Everything is possible :D but to clarify, you can pass anything to the function. Of course in the development Typescript will put a warning and we will catch that in CI. But here the check is for the runtime. Just to be safe ;)